meilisearch/tests/index/stats.rs

```diff
@@ -47,8 +47,8 @@ async fn stats() {
 
 #[actix_rt::test]
 async fn error_get_stats_unexisting_index() {
-    let server = Server::new().await;
-    let (response, code) = server.index("test").stats().await;
+    let server = Server::new_shared().await;
+    let (response, code) = server.unique_index().stats().await;
```

Comment:
```suggestion
    let index = shared_does_not_exists_index()
    let (response, code) = index.stats().await;
```

Hey, I believe you can use the shared index that does not exists to get a known name.

Also, you'll need to edit the name of the index in the `expected_response` below.

Reply:
Awesome, Let me change. 

ya I was thinking about the changes in expected_response and wanted to ask you first 

I am changing it to from

let expected_response = json!({
        "message": "Index not found.",
        "code": "index_not_found",
        "type": "invalid_request",
        "link": "https://docs.meilisearch.com/errors#index_not_found"
    });

to 

let expected_response = json!({
        "message": "Index `DOES_NOT_EXISTS` not found.",
        "code": "index_not_found",
        "type": "invalid_request",
        "link": "https://docs.meilisearch.com/errors#index_not_found"
    });